app/src/main/java/org/mozilla/focus/whatsnew/WhatsNew.kt

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.whatsnew
+
+import android.content.Context
+import android.preference.PreferenceManager
+import android.support.annotation.VisibleForTesting
+import org.mozilla.focus.ext.appVersionName
+
+class WhatsNew {
+    companion object {
+        // TODO: Update to final URL
+        const val SUMO_URL = "https://support.mozilla.org/en-US/products/focus-firefox/firefox-focus-android"
+
+        @VisibleForTesting const val PREFERENCE_KEY_APP_NAME = "lastKnownAppVersionName"
+        private const val PREFERENCE_KEY_SESSION_COUNTER = "updateSessionCounter"
```

Comment:
nit: Should we namespace these preference keys? e.g. is there anything else we might call `updateSessionCounter`?